src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -153,11 +154,16 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="-Wl,--eh-frame-hdr" Condition="'$(_IsApplePlatform)' != 'true'" />
     </ItemGroup>
 
-    <Exec Command="command -v &quot;$(CppLinker)&quot;" IgnoreExitCode="true" StandardOutputImportance="Low">
+    <PropertyGroup>
+      <_CommandProbe>command -v</_CommandProbe>
+      <_CommandProbe Condition="$([MSBuild]::IsOSPlatform('Windows'))">where</_CommandProbe>
```

Comment:
This file is not used on Windows, right? (i.e. we don't support cross-OS publishing in cross build mode)

Reply:
I ran the above `dotnet publish -r linux-bionic-arm64 /p:DisableUnsupportedError=true /p:PublishAotUsingRuntimePack=true` with this on Windows. Worked fine with this change.